guides/customize_tinymce.md

```diff
@@ -73,6 +73,41 @@ That's it. Now the editor can format a block of text as headline 2. Nice!
 Custom buttons always appear in all richtext editors in all elements in Alchemy.
 :::
 
+### Adding editor plugins
+It's possible to configure editor with it's plugins.
+All you have to do is copy (for example) the tinymce list plugin folder to 
```

Comment:
The lists plugin is already part of the bundle shipped with Alchemy. Maybe use another plugin as example or change the text to:


All you have to do is to download the tinymce plugin and copy the folder to: